adobe/spectrum-web-components
# fix(overlay): overlay scroll in safari and firefox

## Description

Overlay needs a `display: flex` applied so that the children render correctly when they overflow the view height and can scroll. 

Resolves overflow and scrolling issues on Firefox and Safari for the following components:
- Overlay
- Combobox
- ActionMenu
- Menu
- Menu with Submenu
- Picker

## Related issue(s)



- https://jira.corp.adobe.com/browse/SWC-609

## Motivation and context

This allows users on Firefox and Safari to scroll and access all children in an overlay 

## How has this been tested?




    1. In Firefox open each component listed above in storybook
    2. Make the viewport short
    3. Content within should scroll and the container should be within the margins
    
    1. In Safari open each component listed above in storybook
    2. Make the viewport short
    3. Content within should scroll and the container should be within the margins

    1. In Chrome open each component listed above in storybook
    2. Make the viewport short
    3. Content within should scroll and the container should be within the margins


## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
